FileLiberator/DownloadPdf.cs

```diff
@@ -1,5 +1,4 @@
-﻿using System;
```

Comment:
Don't be surprised if I leave or re-add system, linq, or generic collections. It's not passive aggressive. These are the base tools I often expect from intellisense